mm/page_alloc.c

```diff
@@ -3281,7 +3281,16 @@ EXPORT_SYMBOL(get_zeroed_page);
 
 void __free_pages(struct page *page, unsigned int order)
 {
+#ifndef CONFIG_KTSAN
 	if (put_page_testzero(page)) {
+#else /* CONFIG_KTSAN */
+	/* Dropping this sync object significantly reduces
+	   the total number of syncs objects (by ~500k),
+	   but can possibly lead to false positives. */
```

Comment:
Add that we did not observe these false positives, otherwise the call looks odd -- we trade memory for false positives. This is not our intention, we don't tolerate false positives.